### PR TITLE
Other resources pagination

### PR DESCRIPTION
**Pull request by alambike**
susceptibles to be paged
- domain
- domain record
- region
- size
- ssh_key
- image action

### CODE REVIEW

**Comment by alambike** [comment]
Sorry, I closed the old PR https://github.com/digitalocean/droplet_kit/pull/54, to clean history. Hope this is good.

File: spec/lib/droplet_kit/resources/image_action_resource_spec.rb [hunk 1]
```diff
@@ -113,6 +113,11 @@ def json
       expect(action.region).to eq(nil)
       expect(action.region_slug).to eq(nil)
     end
+
+    it_behaves_like 'a paginated index' do
+      let(:fixture_path) {'domain_records/all'}
```
Comment (phillbaker) [inline spec/lib/droplet_kit/resources/image_action_resource_spec.rb]: Can you switch this to use https://github.com/digitalocean/droplet_kit/blob/master/spec/fixtures/image_actions/all.json please?

**Comment by phillbaker** [comment]
Looks good other than than the 1 comment, thanks @alambike!

**Comment by alambike** [comment]
Fixed, thanks @phillbaker.

**Pull request merged**